docs/request-options.rst

```diff
@@ -557,9 +557,10 @@ over the wire.
     $clientHandler = $client->getConfig('handler');
     // Create a middleware that echoes parts of the request.
     $tapMiddleware = Middleware::tap(function ($request) {
-        echo $request->getHeader('Content-Type');
+        $contentType = $request->getHeader('Content-Type');
+        echo $contentType[0];
         // application/json
-        echo $request->getBody();
+        echo $request->getBody()->getContents();
```

Comment:
`getContents` will read the "rest of the body". Which is not always what you want. 

`echo $request->getBody();` will work because `getBody` returns a stream that have a `__toString` function.